src/Core/Stock/StockManager.php

```diff
@@ -127,7 +127,7 @@ public function updatePacksQuantityContainingProduct($product, $id_product_attri
 
             // How many packs can be made with the remaining product stocks
             $quantity_by_pack = $pack->pack_item_quantity;
-            $max_pack_quantity = max([0, floor($stock_available->quantity / $quantity_by_pack)]);
+            $max_pack_quantity = max(array(0, floor($stock_available->quantity / $quantity_by_pack)));
```

Comment:
Look weird here, you're right @matthieu-rolland  you need to execute cs fixer

Reply:
done 👍